fpo-web/src/components/FlappSurveys.vue

```diff
@@ -2,14 +2,15 @@
   <div class="fill-body" id="flapp-surveys">
     <main class="app-content">
       <navigation-sidebar />
-      <common-information v-show="getCurrentStepIndex() == 0" v-bind:step="getStep(0)"></common-information>
-      <protection-order v-show="getCurrentStepIndex() == 1" v-bind:step="getStep(1)"></protection-order>
-      <family-law v-show="getCurrentStepIndex() == 2" v-bind:step="getStep(2)"></family-law>
-      <case-management v-show="getCurrentStepIndex() == 3" v-bind:step="getStep(3)"></case-management>
-      <parenting-priority v-show="getCurrentStepIndex() == 4" v-bind:step="getStep(4)"></parenting-priority>
-      <child-relocation v-show="getCurrentStepIndex() == 5" v-bind:step="getStep(5)"></child-relocation>
-      <enforcement-agree v-show="getCurrentStepIndex() == 6" v-bind:step="getStep(6)"></enforcement-agree>
-      <print v-show="getCurrentStepIndex() == 7" v-bind:step="getStep(7)"></print>
+      <get-started v-show="getCurrentStepIndex() == 0" v-bind:step="getStep(0)"></get-started>
```

Comment:
Split the get started step into two steps.
1. Get started step contains the orders list and the protection order questionnaire
2. Common information step contains Your Information and Other Party Information step